is.js

```diff
@@ -129,6 +129,11 @@ var is = function is(x) {
             return parseInt(x.toString().split("").reverse().join("")) == THIRTEEN;
           }
         },
+        atomicNumber: {
+          thirteen: function() {
+            return x.toLowerCase().includes("a","l","u","m","i","u","m");
```

Comment:
Doesn't include the n in "aluminum" nor does it account for "aluminium"

Reply:
@mkem114 
`return x.toLowerCase().includes("a", "l", "u", Math.min("i", "u", "m"));`

Above line appears to be foolproof, it includes the "a", "l", "u" Math.min and "i", "u", "m".

Reply:
sry, now it include the 'n'.

Reply:
> sry, now it include the 'n'.
@mkem114 @elisalter @jezen

Reply:
@sunlee334 Fanttttttttassstic effort my friend. Thamk you.